openverse-api/catalog/api/models/media.py

```diff
@@ -169,6 +146,70 @@ class AbstractMediaReport(models.Model):
     class Meta:
         abstract = True
 
+    def url(self, media_type):
+        url = ('https://search.creativecommons.org/'
```

Comment:
Adding a base_url constant will probably make changing this URL easier when we need to.

Reply:
Addressed in 7e69204.